package.json

```diff
@@ -40,12 +44,13 @@
     }
   },
   "dependencies": {
+    "autoprefixer-loader": "^2.0.0",
     "autoprefixer-core": "^5.2.0",
     "babel": "^5.8.21",
     "babel-loader": "^5.3.2",
     "babel-plugin-typecheck": "^1.2.0",
     "babel-runtime": "^5.8.20",
-    "basscss-sass": "^1.0.2",
+    "basscss-sass": "nicolasiensen/basscss-sass#v1.0.2-branch",
```

Comment:
Should we use latest version? It seems that annoying bug is fixed: https://github.com/basscss/basscss-sass/issues/6#issuecomment-183780817